components/web/nginx/Makefile

```diff
@@ -13,63 +13,62 @@
 #
 
 BUILD_BITS= 64
-# Alas nginx ignores the openssl settings
-OPENSSL_VERSION= 3.1
-USE_PARALLEL_BUILD= yes
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		nginx
-COMPONENT_VERSION=	1.25.4
-COMPONENT_SUMMARY=	Nginx Webserver
-COMPONENT_PROJECT_URL=	https://nginx.net/
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH=	sha256:760729901acbaa517996e681ee6ea259032985e37c2768beef80df3a877deed9
-COMPONENT_ARCHIVE_URL=	https://nginx.org/download/$(COMPONENT_ARCHIVE)
-COMPONENT_SIG_URL=	$(COMPONENT_ARCHIVE_URL).asc
+COMPONENT_NAME= nginx
+COMPONENT_VERSION= 1.25.4
+COMPONENT_SUMMARY= Nginx Webserver
```

Comment:
Please add COMPONENT_REVISION= 1 as we are already on 1.25.4

Reply:
Your PR doesn't change the version but you only found a way to switch to openssl-3.1.
So we agree on the use of COMPONENT_REVISION here as we only change configure options.

Reply:
So, your commit message and the PR's title should also be changed.

Reply:
That happens because we don't use anything to announce work on items. And that'e because we are so few that it would only add bureaucracy and is not worth the efforts.